src/app/window.cpp

```diff
@@ -74,6 +76,7 @@ QTabWidget *MainWindow::tabs_widget()
     launcher->setObjectName("Launcher");
     SettingsTab *settings = new SettingsTab(this);
     settings->setProperty("prevent_disable", true);
+    CameraTab *camera = new CameraTab( this );
```

Comment:
remove spaces around param

Reply:
also would probably make sense to define it in the order they'll appear

Reply:
also set the object name to "Camera"

Reply:
I agree. I put it out of order because this tab uses the settings object. But that is part of the temporary settings thing discussed elsewhere. I will swap back the order when putting the camera logic into the Config object.